### PR TITLE
hangman

### PR DESCRIPTION
**Pull request by kairajohnson**
Comfort: 3.5
Completeness: 2

### CODE REVIEW

File: hangman.js [hunk 1]
```diff
@@ -1 +1,28 @@
...
+document.write ("Hello" + " " + guest + ", Welcome to Hangman!" );
+
+var words = ['love', 'peace', 'hope','happy','life', 'family', 'children', 'unite'];
+
+var guestattempt="";
+
+function chooseWord (words) {
+  var hiddenWord = words[Math.floor(Math.random() * words.length)];
+  prompt (guest + " " + ",Guess My Hangman\n " + hiddenWord )
+If (this.words==guestattempt {
```
Comment (joe-gz) [inline hangman.js:16]: This is missing a closing ")", and make sure you're using lower case for things like "if"

File: hangman.js [hunk 2]
```diff
@@ -1 +1,28 @@
...
+  alert ("Congratulations, your smart after all!"))
```
Comment (joe-gz) [inline hangman.js:17]: I think there is an extra parenthesis here

File: index.html [hunk 3]
```diff
@@ -2,8 +2,12 @@
 <html>
   <head>
     <title>Hangman</title>
+        <link rel="stylesheet" type="text/css" href="hangman.css">
```
Comment (joe-gz) [inline index.html:5]: Do you have a CSS file? if not, I don't think you need to link anything, as it will throw an error

File: hangman.js [hunk 4]
```diff
@@ -1 +1,28 @@
-console.log("welcome to hangman");
\ No newline at end of file
+var guest= prompt('Please enter your Nickname', '');
+if ( (guest=='') || (guest==null) )
+{
+ guest="";
+}
+
+document.write ("Hello" + " " + guest + ", Welcome to Hangman!" );
```
Comment (joe-gz) [inline hangman.js:7]: I like that you've printed something to the browser here!

File: hangman.js [hunk 5]
```diff
@@ -1 +1,28 @@
-console.log("welcome to hangman");
\ No newline at end of file
+var guest= prompt('Please enter your Nickname', '');
+if ( (guest=='') || (guest==null) )
+{
+ guest="";
+}
```
Comment (joe-gz) [inline hangman.js:5]: this if statement probably isnt necessary.  If the prompt is left blank, it will return an empty string regardless.

Comment (kairajohnson) [inline hangman.js:5]: Thanks Joe, I will look over and make corrections this weekend :)

On Fri, Feb 19, 2016 at 1:11 PM, Joe Glatman Zaretsky <
notifications@github.com> wrote:

> In hangman.js
> https://github.com/ga-wdi-exercises/hangman-lab/pull/20#discussion_r53495557
> :
> 
> > \ No newline at end of file
> > +var guest= prompt('Please enter your Nickname', '');
> > +if ( (guest=='') || (guest==null) )
> > +{
> > - guest="";
> >   +}
> 
> this if statement probably isnt necessary. If the prompt is left blank, it
> will return an empty string regardless.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ga-wdi-exercises/hangman-lab/pull/20/files#r53495557.

## 

*LOVE *

_NOTE: This e-mail transmission, including any attachments, is intended
only for the named recipient(s) and may contain information that is
privileged, confidential and/or exempt from disclosure under applicable
law. If you have received this email by intent or accident, you are
forbidden to disclose any information attached or written with anyone
outside of whom this email was intended for. If you have received this
transmission in error, or are not the named recipient(s), please notify
Kaira Johnson kjohnson1913@gmail.com immediately by return e-mail and
permanently delete this transmission, including any attachments._

**Comment by joe-gz** [comment]
Hi Kaira, going forward, if you're feeling like your comfort level with hw is low, please add in some additional comments in the PR letting me know what led to the discomfort.

It feels like there are a few errors in here that should be caught ahead of time.  when I ran the html file, nothing actually happened, as there were already errors holding it up. Did you take a look at the console to see what they were? I've added in some comments here, but I strongly recommend you go back and check the errors for yourself.

In terms of your work flow, I'd avoid doing something complicated like trying to add in randomization until you have everything working. Why not just use one word to start? get the guesses working, and _then_ go back and add in the additional feature of randomization..

Let me know if you want to sit down next week and go through some of this. I'd recommend taking another look at it over the weekend if you can, and really trying to get the basics working.

**Pull request closed without merging**